### PR TITLE
EDSC-2206: Implements collection details

### PR DESCRIPTION
**Pull request by trevorlang**

### CODE REVIEW

File: static/src/js/components/CollectionDetails/CollectionDetailsBody.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+              {urls.dif && buildForDeveloperLink(urls.dif)}
+            </ul>
+          </div>
+          <div className="col-auto">
+            <h4 className="collection-details-body__dev-cat">API Endpoints</h4>
+            <ul className="collection-details-body__dev-list">
+              {urls.granuleDatasource && buildForDeveloperLink(urls.granuleDatasource)}
+              {urls.osdd && buildForDeveloperLink(urls.osdd)}
+              {
+                // TODO: GIBS
```
Comment (mreese84) [inline static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: https://search.earthdata.nasa.gov/search/collection-details?p=C1219032685-LANCEMODIS&m=0.0703125!0!2!1!0!0%2C2&ff=Map%20Imagery has a GIBS link.

Comment (mreese84) [reply static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: And this one has _lots_ of GIBS links -> https://search.earthdata.nasa.gov/search/collection-details?p=C1426751946-LANCEMODIS&m=0.0703125!0!2!1!0!0%2C2&ff=Near%20Real%20Time

Comment (mreese84) [reply static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: OPeNDAP link -> https://search.earthdata.nasa.gov/search/collection-details?p=C191855458-LARC&m=0.0703125!0!2!1!0!0%2C2&q=mopit&ok=mopit

Comment (mreese84) [reply static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: MODAPS link -> https://search.earthdata.nasa.gov/search/collection-details?p=C190733713-LPDAAC_ECS&m=0.0703125!0!2!1!0!0%2C2&q=C190733713-LPDAAC_ECS&ok=C190733713-LPDAAC_ECS

File: static/src/js/util/collectionMetadata/buildUrls.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    id: collectionId,
+    short_name: providedCollectionShortName,
+    version_id: collectionVersionId,
+    is_cwic: isCwic
+  } = json
+
+  const collectionShortName = encodeURI(providedCollectionShortName)
+
+  const cmrMetadataUrlFormats = [
+    { ext: 'html', title: 'HTML' },
```
Comment (trevorlang) [inline static/src/js/util/collectionMetadata/buildUrls.js]: Make sure this url is always visible

File: static/src/js/util/collectionMetadata/buildUrls.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      cmrGranulesUrl = `${apiHost}/granules?url=${encodeURIComponent(cmrGranulesUrl)}&token=${authToken}`
+    }
+
+    urls.granuleDatasource = {
+      title: 'CMR',
+      href: cmrGranulesUrl
+    }
+  }
+
+  // TODO: GIBS
```
Comment (trevorlang) [inline static/src/js/util/collectionMetadata/buildUrls.js]: Get these links in

File: static/src/js/util/focusedCollection.js [hunk 4]
```diff
@@ -12,4 +22,38 @@ const getFocusedCollectionMetadata = (collectionId, collections) => {
...
+    spatial: buildSpatial(ummJson)
+  }
+
+  return {
+    ...jsonMetadata,
+    ...ummJsonMetadata
+  }
+}
+
+export const getCollectionMetadata = async (params, authToken) => {
```
Comment (trevorlang) [inline static/src/js/util/focusedCollection.js]: Think more on this

File: static/src/js/util/scrollParent.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+)
+
+const scrollParent = (node) => {
+  if (!node || node === document.body) {
+    return document.body
+  }
+  return scroll(node) ? node : scrollParent(node.parentNode)
+}
+
+// !node || node === document.body
```
Comment (trevorlang) [inline static/src/js/util/scrollParent.js]: Remove comment

File: static/src/js/util/url/collectionsEncoders.js [hunk 6]
```diff
@@ -100,9 +100,9 @@ export const encodeCollections = (collections, focusedCollection) => {
         excludedGranuleIds = [],
         granules,
         isVisible,
-        metadata
+        isCwic
       } = collection
-      const { is_cwic: isCwic = false } = metadata
+      // const { is_cwic: isCwic = false } = metadata
```
Comment (trevorlang) [inline static/src/js/util/url/collectionsEncoders.js]: Double check comment

**Pull request merged**